desktop/release-notes.md

```diff
@@ -24,13 +24,98 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.16.0
+
+{% include release-date.html date="2022-01-12" %}
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### New
+
+- Extensions have moved from Beta to GA
+- Quick Search has moved from experimental to GA
+- Extensions are now included in Quick Search
+- Analyzing large images is now up to 4x faster
+- New local images view has moved from experimental to GA
+
```

Comment:
```suggestion
- Add Rosetta for Linux as a Beta feature on MacOS 13+ for faster emulation of Intel-based images on Apple Silicon 
```

Reply:
Do we need to say `MacOS 13+`? There is no macOS 14 yet, so probably `MacOS 13` is accurate at this point

Reply:
Shorten a bit to
```
- Rosetta for Linux as a Beta feature on MacOS 13 for faster emulation of Intel-based images on Apple Silicon 
```
Is that all right?

Reply:
Suggestion updated